### DISCUSSION TITLE
Is there a document about DCache?

### DISCUSSION THREAD

**Discussion by ksungkeun84** [chipsalliance/rocket-chip#3661] [Q&A] [answered]
Hi,
I'm trying to analyze the DCache to extend it for my own purpose. It's quite complicated for me to draw a data path out of the source file. Is there any documentation or block diagrams of DCache?

Thanks,

**Comment by astrohan** [comment, marked as answer]
I believe this help you to understand rocket core/cache structure.
https://www.cl.cam.ac.uk/~jrrk2/docs/tagged-memory-v0.1/rocket-core

**Reply by ksungkeun84** [reply]
@astrohan Thanks! I forgot that lowRISC has the block diagram.
